src/content/blog/astro-210.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: "Astro 2.1 Release"
+description: "Introducing core image support • Markdoc integration • Watch mode for astro check • getStaticPaths() TypeScript helpers"
+publishDate: "March 8, 2023"
+authors:
+  - matthew
+coverImage: "/src/content/blog/_images/astro-210/blog-hero-2_1-release.webp"
+socialImage: "/src/content/blog/_images/astro-210/og-image-2_1-release.jpg"
+lang: "en"
+---
+
+import markdoc from "/src/content/blog/_images/astro-210/markdoc.webp"
+import astrocheckwatch from "/src/content/blog/_images/astro-210/astro-check-watch.webp"
+
+import BlogContentImage from "/src/components/BlogContentImage.astro"
+
+We just released Astro 2.1, featuring:
+
+- **[Image optimizations in core](#image-support-in-astro-core)**: An experimental feature that brings the learnings from `@astrojs/image` to core.
+- **[Markdoc integration](#markdoc-integration)**: Markdoc is a new authoring format from Stripe that is optimized to scale to large codebases.
+- **[Watch flag for astro check](#watch-mode-in-astro-check)**: Use this flag to type check your `.astro` files during development.
```

Comment:
Thoughts on renaming this title to `astro check --watch` or `'--watch' support for 'astro check'`?